stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
I don't think that's true at all. We have a pile of common state in cmake/modules/. Having a bit more in the top-level CMakeLists.txt doesn't really make things *that* much worse; no matter what all the real build system would be there.

I'll admit I don't have experience with many CMake-driven projects besides LLVM/Clang, but I don't think they *do* allow you to just build the subdirectory you're in. If they did, there would be less of a reason to ever have CMAKE_SOURCE_DIR, since CMAKE_CURRENT_SOURCE_DIR would be fine.